docs/guides/operator-guide-azure-enclave.md

```diff
@@ -58,12 +58,6 @@ When the registration process is complete, you'll receive the following:
   You'll receive a separate operator key for each deployment environment.
   :::
 
-- A link to the UID2 Private Operator for Azure GitHub release page. For example, it might look something like this: [https://github.com/IABTechLab/uid2-operator/releases/tag/v5.21.5-68a47aec9f-azure-cc](https://github.com/IABTechLab/uid2-operator/releases/tag/v5.21.5-68a47aec9f-azure-cc).
```

Comment:
1) Since we've removed one of the items in this bullet list of two, lines 53-55 say: 

When the registration process is complete, you'll receive the following:

- An operator key, exclusive to you, that identifies you with the UID2 service as a Private Operator. During configuration, use this as the value for `OPERATOR_KEY`. This value is both your unique identifier and a password; store it securely and do not share it.

We don't need a bullet for just one item. Instead, run it on, like this:

When the registration process is complete, you'll receive an operator key,... (etc)

Also, remove the indent off the note tag below, therefore.

2) One other question... is there really nothing they need to do, other than ask the UID2 contact to register their organization? This is titled as a procedure but there's little to be done! Just checking.